### ISSUE TITLE
Combine `Servers Status` and `System Monitors`

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#868]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.4     |
|  Webmin version      |     2.111      |
|  Usermin version     |     2.010     |
|  Virtualmin version  |     7.20.2       |
|  Theme version       |     21.10      |
|  Package updates     |     All installed packages are up to date   |

### background

There are 2 widgets in Virtualmin that do almost the same job (some small differences which I will address below)

- System Monitors (from Webmin)
  ![image](https://github.com/user-attachments/assets/465dab56-9d31-414a-bdc4-b55db0bdafb2)
- Server Status (Virtualmin only)
  ![image](https://github.com/user-attachments/assets/002e4353-b807-4b68-aecc-2f6f51088510)

### the issue

- `System Monitors` are not enabled by default (I thought mine was though)
- The default set of  `System Monitors` is what comes with Webmin and is not really related to the services installed by Virtualmin
- `System Monitors` and `Server Status` have shared functionality that does not need to be replicated
- `System Monitors` does not have a service stop/start/restart button collection

### proposed solution and improvements

-  `System Monitors`
   - should be enabled by default by the Virtualmin installer
   - Virtualmin installer should configure the `System Monitors` that are selected and enabled
   - a service stop/start/restart button collection should be added when a monitor is a service
   - Pre-configured monitors added by Virtualmin could have a flag to prevent them getting deleted but otherwise edited
   - The different PHP versions monitors should be added and removed as they were with `Server` status
   - A watchdog option should be added via checkbox. I know you can add commands to run when the service is discovered to be down but this is not as easy.
   - add an enable/disable option for a monitor, this prevent a user from deleting and monitor to stop it working and would lend itself to the Virtualmin installer configuring mandatory services.
   - ability to change the order of the monitors and this would be reflected in the widget.
- `Server Status`
    - This widget should be retired

### Example default `System Monitors` widget

![image](https://github.com/user-attachments/assets/465ad757-fc5b-4fa8-b0dc-7f3f8b3bc612)

### additional
- https://github.com/virtualmin/virtualmin-gpl/issues/867 - related issue
- https://github.com/webmin/webmin/issues/2233
- https://forum.virtualmin.com/t/system-monitors-vs-servers-status/127101
   -  a lot of suggestions here on how to improve `System Monitors` and possibly combining it with `Server Status`
- https://forum.virtualmin.com/t/can-you-edit-the-servers-in-system-status/126924
  - No, those are hardcoded in Virtualmin.

**Comment by jcameron** [comment]
Yeah I think actually we shouldn't show the Server Status tab by default, at least on Virtualmin installs...

**Comment by jcameron** [comment]
In fact I'm pretty sure it's not on by default! If it is, editing `/etc/webmin/status/config` and changing `sysinfo=1` to `sysinfo=0` should disable it.

**Comment by shoulders** [comment]
I will confirm the tab status on my fresh I stall tommorow, but I think combining them reduces duplication

**Comment by jcameron** [comment]
I don't want to combine them necessarily because systems with only Webmin won't have the "Server Status" widget.

Also, I typed wrong in the comment above - it should be the "System Monitors" widget that is disabled in Virtualmin installs.

**Comment by shoulders** [comment]
> I don't want to combine them necessarily because systems with only Webmin won't have the "Server Status" widget.

I am saying get rid of `Server Status` as it would not be needed with my `proposed solutions` above

> I will confirm the tab status on my fresh I stall tomorrow, but I think combining them reduces duplication

On my fresh install, i can confirm that `System Monitors` are not present

![image](https://github.com/user-attachments/assets/4c8d466a-b04a-4202-b157-059274a45f21)

**Comment by shoulders** [comment]
- @iliaross and @jcameron this one is sent back to you to look at.
- I still believe 100% these 2 modules should be combined as they are performing duplicate functions (certainly a loft of cross-over)

Ref `Add missing services to Server Status Widget` #1190
